### ISSUE TITLE
How to export?

### ISSUE THREAD

**Issue by walter-grace** [theskeletoncrew/treat-toolbox#10]
Is the 'GENERATED COMPOSITE SETS' hash the thing I need or is there a different way to export?

**Comment by skeletoncrewrip** [comment]
yeah, that's a shortcoming at the moment. I have gotten a few requests from folks who are just interested in creating metadata files alongside existing art. definitely have it on my list to make a simple version that doesnt require art to be generative!
